### PR TITLE
Arabic Locale

### PR DESCRIPTION
**Pull request by AhmedNourJamalElDin**

### CODE REVIEW

**Comment by AhmedNourJamalElDin** [comment]
python 2.6 and python 3.3:
```
Downloading archive: https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-2.6.tar.bz2
0.13s$ curl -sSf --retry 5 -o python-2.6.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 404 Not Found
Unable to download 2.6 archive. The archive may not exist. Please consider a different version.
```

**Comment by AhmedNourJamalElDin** [comment]
@hustcc

File: test/testcase.py [hunk 1]
```diff
@@ -700,940 +701,921 @@ def test_timeago_delta(self):
 
     def test_timeago_bg(self):
```
Comment (hustcc) [inline test/testcase.py]: may be we can seperate the test cases by locales.

**Comment by lolobosse** [comment]
@hustcc did you package it?

**Comment by AhmedNourJamalElDin** [comment]
I don't think so cuz in Pypi it's still the old version. please let's package it because I need it badly.

Thanks for your great job, guys. :+1:

**Comment by lolobosse** [comment]
@AhmedNourJamalElDin should be done in `1.0.11`; I just tested it.

**Pull request merged**